CMakeLists.txt

```diff
@@ -40,6 +40,8 @@ endif()
 option(BUILD_TESTING "Enable testing" ON)
 set(VCLIBS_SUFFIX "_oss" CACHE STRING "suffix for built DLL names to avoid conflicts with distributed DLLs")
 
+option(STL_USE_ANALYZE "Pass the /analyze flag to MSVC" OFF)

```

Comment:
Would "Enable static analysis" be more consistent with "Enable testing"?

Reply:
Eh, I like the current message better.